build.gradle.kts

```diff
@@ -61,9 +64,12 @@ repositories {
             password = project.findProperty("gpr.key") as String? ?: System.getenv("GITHUB_TOKEN")
         }
     }
-
-    mavenLocal()
     mavenCentral()
+
+    maven("https://dl.bintray.com/kotlin/ktor")
```

Comment:
Fint om du kan fjerne bintray repoene. De er lagt ned og brukes ikke lenger.
https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

Tror heller ikke at packages.confluent brukes i det prosjektet her da det ikke er noe kafka involvert.

Reply:
Fjerner alt 💣

Reply:
Legg til maven("https://jitpack.io")

Reply:
Ah ja trengte den for å bygge